ajax/libs/livequery/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "livequery",
+  "filename": "jquery.livequery.min.js",
+  "version": "1.1.1",
+  "description": "A jQuery plugin that helps with dynamic DOMs.",
+  "keywords": [
+    "jquery",
+    "dom",
+    "mutation",
+    "event",
+    "observer",
+    "mutationevent",
+    "mutationobserver",
+    "behavior",
+    "plugin",
+    "browser"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brandonaaron/livequery.git"
+  },
+  "author": {
+    "name": "Brandon Aaron",
+    "url": "http://brandon.aaron.sh"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brandonaaron/livequery.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.livequery.*"
```

Comment:
I'll prefer to use `livequery` instead of `jquery. livequery` for the flexibility, and exclude `*.htc` or only include `js`, `css`, and `map` file since there is `jquery.livequery.htc` in master branch which is what we don't need.